osu.Game/Screens/Play/PlayerLoader.cs

```diff
@@ -98,7 +98,11 @@ protected bool BackgroundBrightnessReduction
 
                 backgroundBrightnessReduction = value;
 
-                ApplyToBackground(b => b.FadeColour(OsuColour.Gray(backgroundBrightnessReduction ? 0.8f : 1), 200));
+                ApplyToBackground(b =>
+                {
+                    b.DimWhenUserSettingsIgnored.Value = 0;
```

Comment:
I think that if song select sets this up it should be responsible for cleaning up after itself, rather than having `PlayerLoader` pick this up.

Reply:
Moved to `SongSelect.playExitingTransition()`